data/moves.js

```diff
@@ -10484,6 +10501,7 @@ exports.BattleMovedex = {
 			},
 			onDamagePriority: -101,
 			onDamage: function (damage, target, source, effect) {
```

Comment:
Out of curiosity, how did you get away with typing these parameters? I've been trying to do that for ages because TypeScripting chat commands is hella tedious without it...

Reply:
Oh, I see, `interface` in `globals.ts`... I should have thought of that...